modules/serverless-kafka-source-yaml.adoc

```diff
@@ -59,3 +59,18 @@ spec:
 ----
 $ oc apply -f <filename>
 ----
+
+.Verification steps
+
+You can verify that the Kafka event source was created:
+
+[source, terminal]
+----
+$ oc get pods
+----
+
+[source, terminal]
```

Comment:
Not critical, but consider adding the `.Example output` heading to this example output.